src/HotChocolate/ApolloFederation/src/ApolloFederation/Extensions/SchemaBuilderExtensions.cs

```diff
@@ -0,0 +1,18 @@
+namespace HotChocolate.ApolloFederation.Extensions
```

Comment:
this needs to be moved into the namespace HotChocolate.

Reply:
Moved to namespace HotChocolate